cmake/UserverVenv.cmake

```diff
@@ -95,7 +95,12 @@ function(userver_venv_setup)
 
   set(venv_dir "${parent_directory}/${venv_name}")
   set(venv_bin_dir "${venv_dir}/bin")
-  set("${python_output_var}" "${venv_bin_dir}/python" PARENT_SCOPE)
+  find_program(venv_python
+    NAMES python python3
```

Comment:
We'd use `NO_CACHE`, but it appeared in 3.21

It's easier to check `WIN32` and either use `bin` or `Scripts` based on that

Always using either `python` or `python3` is OK as long as both are present on Windows, choose something that works